HotelManagement/src/tests/RoomTests.java

```diff
@@ -13,4 +13,20 @@ public void setNumberShouldThrowExceptionWhenSettingNegativeRoomNumber() {
 			Room room = new Room(-1);
 		});
 	}
+
```

Comment:
You're testing only the exceptions in here. You should see if the logic you're implementing works fine.

Stick to this pattern for unit tests :

```
public void test() {
//given 
 - What you have in the beggining like parameters and values.
//when
 - calling of methods and changing values. 
//then
  - asserts and checks if you have what you're expecting
}
```